twitterQL/src/db/migrations/20211105151453-create_table_users.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable("users", {
```

Comment:
get used to adding indexes to whatever field you plan to query later (primary keys already have indexes by default)